packages/gpu_voxels/CMakeLists.txt

```
@@ -199,7 +199,7 @@ ICMAKER_CONFIGURE_PACKAGE()
 
 ###############################################################################
 # Build examples
-ADD_SUBDIRECTORY(src/examples)
+# ADD_SUBDIRECTORY(src/examples)
```

I'd suggest adding an `OPTION(BUILD_EXAMPLES ON)` so it doesn't need to be commented
```suggestion
IF(BUILD_EXAMPLES)
  ADD_SUBDIRECTORY(src/examples)
ENDIF(BUILD_EXAMPLES)
```